cmd/juju/backups/restore_test.go

```diff
@@ -149,9 +190,19 @@ func (s *restoreSuite) TestRestoreFromBackupFilenameFail(c *gc.C) {
 }
 
 func (s *restoreSuite) TestRestoreFromBackupId(c *gc.C) {
-	ctlr, apiClient, _ := s.patch(c, nil, nil)
+	ctlr, apiClient, _, modelStatusClient := s.patch(c, nil)
 	defer ctlr.Finish()
+	controllerModelTag := names.NewModelTag(controllerModelUUID)
 	gomock.InOrder(
+		modelStatusClient.EXPECT().ModelStatus(controllerModelTag).Return(
```

Comment:
For these ones where with the same expectations repeated, I created a method to DRY up the tests following @jameinel's suggestion; for example:
https://github.com/juju/juju/blob/17371e7502939753c2dd8505c02b7afd104b12d5/container/lxd/manager_test.go#L281